CredentialProvider.Microsoft/Logging/ConsoleLoggers.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+//
+// Licensed under the MIT license.
+
+using System;
+using System.IO;
+using NuGet.Common;
+
+namespace NuGetCredentialProvider.Logging
+{
+    /// <summary>
+    /// Logs messages to standard output, with log level included
+    /// </summary>
+    internal class StandardOutputLogger : HumanFriendlyTextWriterLogger
+    {
+        public StandardOutputLogger() : base(Console.Out) { }
+    }
+
+    /// <summary>
+    /// Logs messages to standard error, with log level included
+    /// </summary>
+    internal class StandardErrorLogger : HumanFriendlyTextWriterLogger
+    {
+        public StandardErrorLogger() : base(Console.Error) { }
+    }
+
+    /// <summary>
+    /// Emits a log message in a human-readable format to a TextWriter, including the log level
+    /// </summary>
+    internal class HumanFriendlyTextWriterLogger : LoggerBase
```

Comment:
FYI FileLogger could also use this (currently it does its own formatting), but didn't want to make that change yet.